app/term/term.html

```diff
@@ -157,6 +157,9 @@ <h3 class="term-section-header"><span uib-tooltip-template="'childterms.html'" t
             <td>{{childRelation.typeOf}}</td>
           </tr>
         </table>
+        <a ng-show="childTermsPageSize < termModel.childTermsRelations.length" ng-click="childTermsPageSize = termModel.childTermsRelations.length" class="glyphicon glyphicon-collapse-down">show more</a>
+        <a ng-show="(childTermsPageSize == termModel.childTermsRelations.length && termModel.childTermsRelations.length > defaultPageSize)" ng-click="childTermsPageSize = defaultPageSize" class="glyphicon glyphicon-collapse-up">show less</a>
+
```

Comment:
The font looks a bit odd. Some space between the icon and the text would be better, and that much space between words. Also, the dotted underlined looks too close to the text.